packages/grid/scss/_col.scss

```diff
@@ -13,7 +13,7 @@
 @import '@carbon/layout/scss/breakpoint';
 @import 'prefix';
 
-@mixin make-col-ready($gutter: $grid-gutter) {
+@mixin carbon--make-col-ready($gutter: $carbon--grid-gutter) {
```

Comment:
Per the conversation on slack, I thought the `carbon` name space was for mixins and functions only so I'm confused as to why the parameter default variable is prefixed as well.

Reply:
@elizabethsjudd apologies for the confusion, unfortunately variables need to be a part of this namespace PR in order to prevent collisions. We've had issues with things like `$breakpoints` overlapping, for example, that this helps to address. Let me know if there is a good way to circumvent this! Happy to drop namespaces on variables 😄

Reply:
Either way I just feel like things need to be consistent.  From what I've seen we have:
* non-prefixed variables
* `carbon` prefixed variables
* `carbon` prefixed mixins and functions
* and `bx` prefixed classes

If I was new to the system and just reading the code, I would think that carbon was a mixture of 3 different systems.  I understand that this may be out of scope for Carbon X but something to consider for Carbon 11.  Typically these types of breaking changes are fairly easy to resolve (find and replace) and I'd rather have to update variable names than have confusing code.

As a developer who actually has to call these mixins within my system as well, I still feel like `carbon--` is a lot to type for each and every mixin.  The `rem` mixin is a really good example.  I use it all the time in PAL sass (it's very helpful! 🎉 ) but it's more than tripled in length. I'm a strong believer in the fact that namespaces should be as short as possible. Using short namespaces help to not draw focus from the function/mixin name (the important part) which makes the code scannable while maintaining scoping and understandability that code belongs to a single system.

Reply:
@elizabethsjudd RE: consistency, I made this note in the PR which I believe addresses this:

> Add carbon-- namespace to public Sass API.

The rule of thumb is that variables, mixins, and functions that are considered public are namespaced. We can definitely change that, but I believe under that assumption things should be considered consistent here (or at least that's the intent).

RE: selectors, definitely understand the prefix frustration. For folks using Sass, we suggest that instead of hard-coding `bx` that they use the `$prefix` variable from Carbon. We would love to switch the selector, but unfortunately the naming decision just hasn't lined up for this the way we wanted to.

Reply:
I understood when you added the namespace in the PR because of that comment, but it's not documented anywhere.  I'm talking from a new user who is looking at code, are they going to understand when it's used? 

As far as using `$prefix` that only works if you're actually in the Sass.  If you are not using one of the pre-defined Carbon frameworks, the class names are hard coded in to that system. Again, this all comes down to being new users to Carbon in a variety of different setups.  It's not clear around where, when, and how these name spaces are used.

Reply:
@elizabethsjudd thanks for the feedback! Definitely agreed this needs documentation 👍 Thankfully this will be the focus in our beta release as things become stabilized. Will also always accept PRs that help to improve docs 😄